internal/databases/mongo/binary/mongod.go

```diff
@@ -131,11 +131,11 @@ func (mongodService *MongodService) GetBackupCursor() (cursor *mongo.Cursor, err
 		if !backupCursorErrorIsRetried(err) {
 			return nil, err
 		}
-		if i < cursorCreateRetries {
-			minutes := time.Duration(i + 1)
-			tracelog.WarningLogger.Printf("%v. Sleep %d minutes and retry", err.Error(), minutes)
-			time.Sleep(time.Minute * minutes)
-		}
+
+		// logging retries
+		minutes := time.Duration(i + 1)
+		tracelog.WarningLogger.Printf("%v. Sleep %d minutes and retry", err.Error(), minutes)
+		time.Sleep(time.Minute * minutes)
```

Comment:
looks like with `i >= cursorCreateRetries` we need to log only error, without message about new retry and sleep (as we won't retry anymore)